packages/react-router-dom/docs/guides/query-string-parsing.md

```diff
@@ -0,0 +1,61 @@
+# Query String Parsing
+
+In earlier versions of React Router we provided out-of-the-box support for query string parsing. Since there are too many different parsing-algorithms out there and people had problems with an opinionated version, it's been removed from React Router since v4.
```

Comment:
Thanks for the detailed explanation! So, shouldn't we explain the *why* at all?
Or just summarize it shortly? Like:
> Since there's no defined standard for parsing query strings, React Router is not opinionated  here and leaves the implementation up to you.